src/ISA/ISA/ArcTypes/CompositeColumn.fs

```diff
@@ -48,4 +48,21 @@ type CompositeColumn = {
                     let _, unit = cell.AsUnitized
                     unit
         |]
-        if Array.isEmpty arr then None else Some arr
\ No newline at end of file
+        if Array.isEmpty arr then None else Some arr
+
+    /// <summary>
+    /// Simple predictor for empty default cells.
+    /// </summary>
+    member this.PredictNewColumnCell() =
```

Comment:
`Predict` is not the correct term here. How about something in the lines of `GetDefaultEmptyCell`?